Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 55;
```

Comment:
Shouldn't be in there

Reply:
Will remove but it will probably keep coming...